src/components/layout/ContainerWrapper.vue

```diff
@@ -13,5 +13,12 @@
 <style lang="scss" scoped>
 .container-xxl {
   max-height: 100vh;
+  overflow-y: auto;
+  &::-webkit-scrollbar {
```

Comment:
this just erase the scrollbar, but the issue persists, that is something I will fix in my next pr, by now I', just gonna let this go